NEWS.md

```diff
@@ -1,3 +1,7 @@
+# development version
+
+* export `ud_are_convertible` , #258 @cregouby
```

Comment:
Please, add this to the section below (version 0.7-0), which is the next release we are preparing.

Reply:
Also, add the PR:

```
* export `ud_are_convertible`; #263 addressing #258, @cregouby
```